server/src/jobs/jobs.ts

```diff
@@ -71,6 +72,9 @@ export const CronsMap = {
       // # Remplissage des réseaux
       await addJob({ name: "hydrate:reseaux", queued: true });
 
+      // # Remplissage des ofa inconnus
+      await addJob({ name: "hydrate:ofa-inconnus", queued: true });
```

Comment:
Si je comprends bien les roganismes seront remis chaque jour via référentiel, puis mis à jour via l'api entreprise. J'imagine que ça représente des gros volumes. Ça prend beaucoup de temps ce script (rien de bloquant, juste par curiosité)

Reply:
C'est un peu long au tout début (20-30min), mais ensuite vu qu'on aura quasiment plus d'OF avec enseigne ou raison sociale manquante, ca sera très rapide.

Après qqs tests en local ca donne ca : 

- 1er lancement 30 et qqs minutes 
- 2e lancement il ne reste que les OFA pour lesquels le siret est un faux ou non trouvé dans l'API donc très rapide

Reply:
Mais c'est pas recalculé chaque nuit via MàJ du référentiel ?

Reply:
Alors bonne remarque.

🤔 normalement chaque nuit on fait un insertOrUpdate pour les OF du référentiel.
Donc si l'OF à déja une raison sociale / enseigne (qui a été corrigée la veille), on ne devrait pas le toucher en updatant depuis le référentiel.

Cf la fonction insertOrUpdateOrganisme dans le job hydrate-organisme.ts

```
// Update de l'organisme sans appels API si existant
    const updatedOrganisme = {
      ...organismeInTdb,
      ...(raison_sociale ? { nom: raison_sociale } : {}),
      ...(raison_sociale ? { raison_sociale } : {}),
      ...(enseigne ? { enseigne } : {}),
      nature: nature,
      adresse: adresseFormatted,
      ferme: isFerme,
      qualiopi: qualiopi || false,
      fiabilisation_statut:
        !isFerme && uai ? STATUT_FIABILISATION_ORGANISME.FIABLE : STATUT_FIABILISATION_ORGANISME.INCONNU,
      est_dans_le_referentiel: uaiMultiplesInTdb
        ? STATUT_PRESENCE_REFERENTIEL.PRESENT_UAI_MULTIPLES_TDB
        : STATUT_PRESENCE_REFERENTIEL.PRESENT,
    };
```